src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/AbstractExpressionHandler.java

```diff
@@ -176,29 +176,6 @@ protected final boolean isOnStartOfLine(DetailAST ast) {
         return getLineStart(ast) == expandedTabsColumnNo(ast);
     }
 
-    /**
-     * Searches in given sub-tree (including given node) for the token
-     * which represents first symbol for this sub-tree in file.
-     *
-     * @param ast a root of sub-tree in which the search should be performed.
-     * @return a token which occurs first in the file.
-     * @noinspection WeakerAccess
-     */
-    public static DetailAST getFirstToken(DetailAST ast) {
-        DetailAST first = ast;
-        DetailAST child = ast.getFirstChild();
-
-        while (child != null) {
-            final DetailAST toTest = getFirstToken(child);
-            if (toTest.getColumnNo() < first.getColumnNo()) {
-                first = toTest;
-            }
-            child = child.getNextSibling();
-        }
-
-        return first;
-    }
-
```

Comment:
This method is completely necessary now that we can simply get the type of method/ field as first child of `TYPE` ast.